test/test-tab.js

```diff
@@ -127,12 +127,6 @@ exports["test behavior on close"] = function(assert, done) {
                      "After being closed, tab attributes are undefined (url)");
         assert.equal(tab.index, undefined,
                      "After being closed, tab attributes are undefined (index)");
-        if (app.is("Firefox")) {
-          // Ensure that we can call destroy multiple times without throwing;
-          // Fennec doesn't use this internal utility
-          tab.destroy();
-          tab.destroy();
-        }
```

Comment:
I'm not sure what the fennec comment is about.  The reason this test was added as because calling `tab.destroy();` previously didn't clean itself up properly and it was only discovered because someone pointed out calling destroy on a tab twice was producing an error.  Alex fixed this one, and we determined that part of the test should include calling destroy more than once, to ensure that does not produce an error.  So I think we should keep this, I don't see any reason to change it yet anyhow, I don't understand how it could be a footgun.